test/test_surface_fluxes.jl

```diff
@@ -120,7 +120,7 @@ _fractional_indices(at_node, grid, ::Nothing, ::Nothing, ::Nothing) = (nothing,
             # mid-latitude ocean conditions
             set!(ocean.model, u = 0, v = 0, T = 15, S = 30)
             
-            coupled_model = OceanSeaIceModel(ocean; atmosphere, similarity_theory)
+            coupled_model = OceanSeaIceModel(ocean; atmosphere, ocean_similarity_theory=similarity_theory)
```

Comment:
This doesn't make sense to me. It's a similarity theory for the atmosphere, not for the ocean.